Sources/Common/Background Queue/QueueRunRequest.swift

```diff
@@ -95,7 +95,7 @@ public class CioQueueRunRequest: QueueRunRequest {
 
                 // The task failed to execute like a HTTP failure. Update `lastFailedTask`.
                 updateWhileLoopLogicVariables(didTaskFail: true, taskJustExecuted: nextTaskToRunInventoryItem)
-                break
+                continue // quit loop early and move to next task.
```

Comment:
Does this possibly fix the [bug that the customer reported](https://github.com/customerio/issues/issues/11546) ?